src/plugins/ini/ini.c

```diff
@@ -79,10 +80,36 @@ static int iniKeyToElektraKey (void *vhandle, const char *section, const char *n
 		{
 			keyAddBaseName(appendKey, section);
 		}
+		Key *sectionKey = ksLookup(handle->result, appendKey, KDB_O_NONE);
+		if(sectionKey)
+		{
+			if(!(keyGetMeta(sectionKey, "ini/index")))
```

Comment:
there were, and probably will be in future when i can figure out how to do it the way i wanted to. 
i planned on using 3 order keys (keyIndex, sectionIndex and subKeyIndex(keys below a section)) but couldn't manage to get it working yet. but i'll use `order` for now